apps/nova/src/lib/room.ts

```diff
@@ -1,18 +1,45 @@
-import { IRoom } from "@litespace/types";
+import { IRoom, IUser } from "@litespace/types";
+
+type OtherMember = {
+  id: number;
+  name: string | null;
+  image: string | null;
+  role: IUser.Role;
+  lastSeen: string;
+  online: boolean;
+  gender: IUser.Gender | null;
+};
+
+export type RoomsMap = { [roomId: number]: { [userId: number]: boolean } };
```

Comment:
I believe this type should exported be from headess. 

There is a problem with this type as well. **It is unsafe**. If you tried to access any room or any user id, you will not get any typescript error (unsafe). This makes it prone to errors. Please make it type-safe and error-proof. 



```suggestion
export type RoomsMap = { [roomId: number]?: { [userId: number]?: boolean } };
```